detox/android/detox/src/full/java/com/wix/detox/espresso/EspressoDetox.java

```diff
@@ -120,7 +120,7 @@ public static void setURLBlacklist(final ArrayList<String> urls) {
         UIThread.postSync(new Runnable() {
             @Override
             public void run() {
-                NetworkIdlingResource.setURLBlacklist(urls);
+                NetworkIdlingResource.Companion.setURLBlacklist(urls);
```

Comment:
Can't we use `@JvmStatic` to clean this up to:
```suggestion
                NetworkIdlingResource.setURLBlacklist(urls);
```